Documentation/certificate-generation-resources.md

```diff
@@ -19,8 +19,8 @@ Parameters are passed to/from the endpoints as JSON key-value pairs.
 
 ## Settings on Bridge initialization
 
-* `BridgeCertificatePassword` - string - password for certificates generated/exported. Default is "test"
-* `BridgeCertificateValidityPeriod` - TimeSpan - valid timespan for certificates generated. Default is 24 hours
+* `TestRootCertificatePassword` - string - password for certificates generated/exported. Default is "test"
```

Comment:
As discussed f2f - I prefer to deprecate them later; even though these changes are not necessary for testing in the new infrastructure. We maintain it until we get rid of it by deleting it :smile: